forgerock-authenticator/src/main/java/org/forgerock/android/auth/AuthenticatorManager.java

```diff
@@ -171,6 +171,19 @@ boolean removeNotification(PushNotification notification) {
         return storageClient.removeNotification(notification);
     }
 
+    PushNotification getNotification(String notificationId) {
+        Logger.debug(TAG, "Retrieving PushNotification with ID '%s' from the StorageClient.", notificationId);
+
+        // Retrieve notification from StorageClient
+        PushNotification notification = storageClient.getNotification(notificationId);
```

Comment:
What if notificationId not found? should return null instead of throw NPE?

Reply:
It will return null